src/components/Events/EventsList.vue

```diff
@@ -1,18 +1,14 @@
 <template>
     <div>
-        <span v-if="noEvents">
-          <span v-if="myEvents">You aren't registered for any events.</span>
-          <span v-else>There aren't any events right now. Check back later!</span>
-        </span>
+        <span v-if="noEvents">{{ noEventsMsg }}</span>
```

Comment:
i think that you should keep the previous logic because there is a disctinction between there being no events and not being registered for any. That also means you dont have to pass in the noEvents message as a prop in Events.vue

Reply:
I didn't actually write this code (I think it was from Bill's abstraction of the events list?), but I think it got changed later anyways.